modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1180,8 +1168,8 @@ private RaftGroupEventsListener createRaftGroupEventsListener(int zoneId, TableP
                 replicaGrpId,
                 busyLock,
                 partitionMover,
-                rebalanceScheduler,
-                zoneId
```

Comment:
Method parameter zoneId is no longer used.

Reply:
fixed